docs/clients/rgb_led.rst

```diff
@@ -0,0 +1,132 @@
+
+Arduino
+~~~~~~~
+
+To use the RGB LED Interface in Arduino, ensure rgb_led.hpp is included. This
+allows the creation of a RgbLedClient object. See the Message Table below for available messages. 
+All message objects use the Short Name with a trailing underscore. 
+All messages use the standard Get/Set/Save functions. 
+
+A minimal working example for the RgbLedClient is:
+
+.. code-block:: Arduino
+
+    #include <iq_module_communication.hpp>
+    
+    IqSerial ser(Serial2);
+    RgbLedClient rgbInterface(0);
+    
+    void setup() {
+        ser.begin();
+        // Initialize Serial (for displaying information on the terminal)
+        Serial.begin(115200);
+    }
+    
+    void loop() {
+        int redIntensity = 0;
+        if(ser.get(rgbInterface.red_, redIntensity))
+        Serial.println(redIntensity);
+    }
+
+C++
+~~~
+
+To use the RGB LED Interface client in C++, include rgb_led.hpp. This allows the
+creation of an RgbLedClient object. See the Message Table below for available messages. All message objects
+use the Short Name with a trailing underscore. All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the RgbLedClient is:
+
+.. include:: ../notes/c-full-code.rst
+
+.. code-block:: c++
+
+    #include "generic_interface.hpp"
+    #include "rgb_led.hpp"
+
+
+    void main(){
+        // Make a communication interface object
+        GenericInterface com;
+
+        // Make an RGB LED Interface object with obj_id 0
+        RgbLedClient rgbInterface(0);
+
+        // Use the RGB LED Interface Client
+        rgbInterface.red_.get(com)
+
+        // Insert code for interfacing with hardware here  
+    }
+
+Matlab
+~~~~~~
+
+To use the RGB LED Interface client in Matlab, all Vertiq communication code must be included in your
+path. This allows the creation of a RgbLedClient object. See the Message Table below for available messages.
+All message strings use the Short Names. All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the RgbLedClient is:
+
+.. code-block:: Matlab
+
+    % Make a communication interface object
+    com = MessageInterface(’COM18’,115200);
+
+    % Make a RgbLedClient object with obj_id 0
+    RgbInterface = RgbLedClient(’com’,com);
+
+    % Use the RgbLedClient object
+    red = RgbInterface.get(’red’);
+
+Python
+~~~~~~
+
+To use the RGB LED Interface Client in Python, import ``iqmotion`` and create a module that has the RGB LED Interface Client within its firmware. 
+See the table below for available messages. All message strings use the Short Names. 
+All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the RGB LED Interface Client is:
+
+.. code-block:: Python
+    :substitutions:
+    
+    import iqmotion as iq
+
+    com = iq.SerialCommunicator("/dev/ttyUSB0")
+    |variable_name| = iq.|module_name|(com, 0, firmware="speed")
```

Comment:
@vertiq-jordan Any chance you could take a quick look at it and see if you can get any idea what the cause is? If its something simple, it would be nice to resolve it here

Reply:
I think it's a remnant of changing things over to a new style of RTD. To fix it now, we'll need to make a file that defines the names (preferably one that works for all clients...somehow), and include that file everywhere...so, not a small change. Also begs the question for the python api...can we have a module called advancedspeed that works with all advanced speeds? There's no good way to document one that will work for everything.